.gitignore

```diff
@@ -132,4 +132,7 @@ dmypy.json
 .vagrant
 
 # Database
-db.sqlite3
\ No newline at end of file
+db.sqlite3
+
+#idea
+.idea
```

Comment:
Please use this to avoid adding files implicitly.
.idea/
And remove all other files under this directory

Please make sure all other intellij sdk are not in the PR